lib/transactions.js

```diff
@@ -0,0 +1,44 @@
+import { pickRandomNumber } from './random.js'
+
+/**
+ * Fetches a random list of confirmed transaction IDs from the Arweave network.
+ *
+ * @param {Function} fetchTransactions - A function to fetch transactions from a specific page.
+ * @returns {Promise<string[]>} A promise that resolves to an array of confirmed transaction IDs.
+ */
+export const getTransactions = async (fetchTransactions = fetchTransactionsFromViewblock) => {
+  // We hardcode the min and max page numbers for now to avoid hitting the API too much
+  // Transactions before page 50 are less likely to be confirmed
+  const minPage = 50
+  // Max page set to 10,000 to fetch recent transactions. Adjust as needed (max is around 2.8M).
+  const maxPage = 10_000
+
+  // TODO: Find a publicly documented API
+  const randomPageNumber = pickRandomNumber(minPage, maxPage)
```

Comment:
A publicly documented API for what?

This way it looks like you want an API for picking a random number 😉 

Please add more details to the code comment.

Reply:
Woops, this comment was meant to be placed bellow where we are actually trying to fetch transactions. Will move it and add more context.